### DISCUSSION TITLE
How do you make a dynamic? list in quasar?

### DISCUSSION THREAD

**Discussion by linomass36** [quasarframework/quasar#12653] [General - Components / Directives / etc] [answered]
So I don't know how you call this, but I want to make a list in quasar, where the data is inputted by the user and that adds a row with there input.

I've been working with quasar for about a month know, but I'm still pretty knew to most of it. I've already seen examples of todo lists, but that layout is that you input your data eg. at the top of the page and that creates a row based on that, but I want a row where you input your data and when you press enter, it adds a new blank row (from a template) underneath the now filled in one.

**Comment by MilosPaunovic** [comment, marked as answer]
I think this is something in the lines of what you're going for:
[https://codepen.io/milospaunovic/pen/QWOYrXR?editors=1011](https://codepen.io/milospaunovic/pen/QWOYrXR?editors=1011)

Next time, please do try it yourself first and come with a specific problem or the code segment which got you stuck, so it's not like someone is doing all the work for you.

**Reply by linomass36** [reply]
Oh, yeah sorry. didn't mean it like that. I'm new to programming and i didn't know how to approach this problem at all
